dwitter/views_v2.py

```diff
@@ -82,7 +83,17 @@ class DweetViewSet(mixins.RetrieveModelMixin,
     serializer_class = DweetSerializer
 
     def retrieve(self, request, pk):
-        self.queryset = self.queryset.annotate(
+        # For single dweet view, also include deleted dweets
```

Comment:
Why? 🤔

Reply:
For two reasons:
1. Feature parity with previous frontend, which shows you a [deleted dweet] page rather than a 404 for deleted dweets
2. To allow you to follow remix chains through deleted dweets.

I'm happy to have the discussion of whether we should permanently delete all records of the dweets instead,  but that would break remix chains .